app/configurations/config.okc.js

```diff
@@ -83,6 +83,7 @@ export default configMerger(walttiConfig, {
   viaPointsEnabled: true,
   includeCarSuggestions: false,
   includeParkAndRideSuggestions: false,
+  minutesToDepartureLimit: 360,
```

Comment:
IMO `departuresUseRelativeTimeUpTo` or `departuresRelativeTimeLimit` would be more understandable.

Reply:
I agree. The name is the one used by DT (see `DepartureRow`), so I did not want to change it. We should suggestion this change upstream.